CHANGELOG.md

```diff
@@ -7,6 +7,103 @@ players.
 
 ## [0.9]
 
+### [0.9.0]
+
+#### Release summary
+
+TODO - write info about the release comparing with 0.8.2.
+
+Summary:
+- kube-lego broke during 0.8.2, and is now replaced
+- ...
+
+Bumped dependencies:
+- jupyterhub version 1.1.0
+- kubespawner bump
+- oauthenticator bump
+- chp bump
+- ...
```

Comment:
A few more dependencies and the version they've been bumped to.
```suggestion
- jupyterhub version 1.1.0
- configurable-http-proxy version  4.2.1
- jupyterhub-ldapauthenticator version 1.3.0
- jupyterhub-kubespawner version 0.11.1 
- oauthenticator version 0.11.0
- kubernetes version 10.0.1
- mwoauth version 0.3.7
- cryptography version 2.8.*
```

Reply:
Thank you @GeorgianaElena!